ts/packages/anchor/package.json

```diff
@@ -33,8 +33,8 @@
     "test": "jest tests --detectOpenHandles"
   },
   "dependencies": {
-    "@project-serum/borsh": "^0.2.5",
-    "@solana/web3.js": "^1.64.0",
+    "@coral-xyz/borsh": "./packages/borsh",
```

Comment:
Anchor is a separate npm package, so you need to include `coral-xyz` as a dependency for the Anchor package. I'd say you can remove this line and do `yarn add @coral-xyz/borsh` within the `ts/packages/anchor` folder.